README.md

```diff
@@ -72,75 +72,108 @@ For guidance on developing for PowerToys, please read the [developer docs](/doc/
-- All PowerToys registry entries are moved from machine scope (HKLM) to user scope (HKCU).
-- Quick access system tray launcher. Thanks [@niels9001](https://github.com/niels9001)!
+- New utility: Paste as Plain Text allows pasting the text contents of your clipboard without formatting. Note: the formatted text in the clipboard is replaced with the unformatted text. Thanks [@carlos-zamora](https://github.com/carlos-zamora)!
+- New utility: Mouse Jump allows to quickly move the mouse pointer long distances on a single screen or across multiple screens. Thanks [@mikeclayton](https://github.com/mikeclayton)!
+- Add new GPO policies for automatic update downloads and update toast notifications. Thanks [@htcfreek](https://github.com/htcfreek)!
+- Support MSC and CPL files in "Run command" results of PowerToys Run Program plugin. Thanks [@htcfreek](https://github.com/htcfreek)!
+- Add support for log2 and log10 in PowerToys Run Calculator plugin. Thanks [@RickLuiken](https://github.com/RickLuiken)!
+- Added experimentation to PowerToys first run experience.  There are current page which says "welcome" and a variant with direct instructions on how to use some of the utilities.  We want to see if directly showing how to use PowerToys leads to more people using the features :) 
 
-### Awake
+### General
 
-- Disable instead of hiding "Keep screen on" option.
+- Improve metered network detection in runner. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
+- Update PowerToys logo used by installer. Thanks [@ChaseKnowlden](https://github.com/ChaseKnowlden1)!
+- Add new GPO policies for automatic update downloads and update toast notifications. Thanks [@htcfreek](https://github.com/htcfreek)!
+- Update copyright year to 2023. Thanks [@ChaseKnowlden](https://github.com/ChaseKnowlden)!
 
 ### FancyZones
 
 - Refactored and improved code quality.
+- Fix crashing on moving window between monitors with Win + arrows. (This was a hotfix for 0.67)
+- Fix issue causing window attributes to not be reset properly. (This was a hotfix for 0.67)
+- Fix issue causing window to not be adjusted when layout is changed. (This was a hotfix for 0.67)
+- Fix issue causing window not to be unsnapped on drag started. (This was a hotfix for 0.67)
+- Fix issue causing layouts not to be applied to new virtual desktops. (This was a hotfix for 0.67)
+- Fix issues causing windows not to be restored correctly to their last known zone.
 
 ### File explorer add-ons
 
-- Fixed escaping HTML-sensitive characters when previewing developer files. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
+- Add Developer files previewer option to set max file size and fix styling issue. Thanks [@Aaron-Junker](https://github.com/Aaron-Junker)!
+- Improve Developer files previewer exception handling and printing of error messages.
+- Fix crash when generating PDF and Gcode files thumbnails. (This was a hotfix for 0.67)
```

Comment:
```suggestion
- Fix crash when generating PDF and Gcode file thumbnails. (This was a hotfix for 0.67)
```

Don't we need "file" in this case as singular because we have thumbnails as plural? 🤔

Reply:
"pdf and gcode" implies plural too. Better would be: "...thumbnails for pdf and gcode files"